msal_extensions/windows.py

```diff
@@ -109,7 +109,7 @@ class WindowsTokenCache(msal.SerializableTokenCache):
     """
     def __init__(self,
                  cache_location=os.path.join(
-                     os.getenv('LOCALAPPDATA'),
+                     os.getenv('LOCALAPPDATA', ''),
```

Comment:
Interesting. With hindsight, env variable "LOCALAPPDATA" would probably not exist on Linux and macOS either? In such case, the default value would write into a file named ".IdentityService/msal.cache" in current app's cwd, which defeats the purpose of sharing token across apps possibly developed in different languages.

We could consider using something [like this](https://docs.python.org/3/library/os.path.html#os.path.expanduser). Please test it, and also bring this up to .Net extension folks.

```python
os.getenv('LOCALAPPDATA', os.path.expanduser('~'))
```

Reply:
Fixed with the combination of 934ef7d and 3da5724

Reply:
The fix looks good. I'm closing this thread. Once you get the remaining Travis CI errors fixed, we would be able to wrap up this.